sky/serve/load_balancer.py

```diff
@@ -223,10 +229,25 @@ async def startup():
         uvicorn.run(self._app, host='0.0.0.0', port=self._load_balancer_port)
 
 
-def run_load_balancer(controller_addr: str, load_balancer_port: int):
-    load_balancer = SkyServeLoadBalancer(controller_url=controller_addr,
-                                         load_balancer_port=load_balancer_port)
-    load_balancer.run()
+def run_load_balancer(controller_addr: str,
+                      load_balancer_port: int,
+                      policy_name: Optional[str] = None):
```

Comment:
```suggestion
                      load_balancing_policy_name: Optional[str] = None):
```
Lets align the argument name